compsets/hera.input

```diff
@@ -17,7 +17,7 @@ platform hera.intel {
     cores_per_node=40 # Number of cores per node on compute nodes
     cpus_per_core=2
     C768_PPN=10
-    C768_THRD=4
+    C768_THRD=1
```

Comment:
Can we increase C768_PPN to 40 now that C768_THRD is 1?

Reply:
No, WW3 still needs the extra memory per core. The option to use C768_THRD=1 is for ensuring the output is b4b reproducible in successive runs until we find out how to make WW3 thread-safe. I hope that is acceptable.